### DISCUSSION TITLE
Perfect Usb Cable for Flipper !

### DISCUSSION THREAD

**Discussion by Foul** [djsime1/awesome-flipperzero#43] [Ideas]
Hi !
I think you can add this Perfect usb cable for Flipper in "Modules & Cases" section :

mumbi 26110 Cable USB A to Cable USB C
https://www.amazon.fr/gp/product/B074TGRPL7

:+1:

**Comment by djsime1** [comment]
Couple of issues:
1. It's out of stock
2. It's only on Amazon's French website
3. I'd perfer to not put paid accessories on this list
3.5 (With the exception of DIY modules as already present)
For those reasons, I won't be adding this to the list; Sorry.

**Reply by Foul** [reply]
I'm french and i understand..no problem ;)

**Comment by CLStan** [comment]
Try this one then...

ApexOne USB C Charger Cable, Portable Keychain 0.75ft Short Type-C Charging & SYNC Nylon Braid Cord, Compatible with Samsung S9 S8 Note 9 8 LG V30 G6 Oneplus https://a.co/d/cQJCR7j
